apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx

```diff
@@ -1,9 +1,149 @@
 import { PatientSearchResult } from 'generated/graphql/schema';
+import styles from './PatientSearchResultListItem.module.scss';
+import { Link } from 'react-router-dom';
 
 type Props = {
     result: PatientSearchResult;
 };
 
-const PatientSearchResultListItem = ({ result }: Props) => <>{result.patient}</>;
+const PatientSearchResultListItem = ({ result }: Props) => {
+    const { legalName, birthday, gender, shortId, phones, names, emails, addresses, identification } = result;
+    const noData = 'No Data';
+    const driversLicense = identification.find((id) => id.type === "Driver's license number");
+    const ssn = identification.find((id) => id.type === 'Social Security');
+    const first = legalName?.first ?? '';
+    const last = legalName?.last ?? '';
+    const firstName = first.charAt(0).toUpperCase() + first.slice(1).toLowerCase();
+    const lastName = last.charAt(0).toUpperCase() + last.slice(1).toLowerCase();
+
+    return (
+        <div className={styles.listItem}>
+            <div className={styles.listItemBox}>
+                <div className={styles.listItemData}>
+                    <label htmlFor="legalName" className={styles.listItemLabel}>
+                        LEGAL NAME
+                    </label>
+                    <br />
+                    <Link
+                        id="legalName"
+                        className={`${styles.value}, ${styles.name}`}
+                        to={`patient-profile/${result.patient}/summary`}>
```

Comment:
praise:  Much better approach than the current `PatientResult` component that uses a function